src/api/app/views/webui/request/beta_show.html.haml

```diff
@@ -76,3 +76,16 @@
           = render partial: 'webui/request/beta_show_tabs/changes', locals: { bs_request: @bs_request, action: @action, refresh: @refresh }
         .tab-pane.fade.p-2#mentioned-issues{ 'aria-labelledby': 'mentioned-issues-tab', role: 'tabpanel' }
           = render partial: 'webui/request/beta_show_tabs/mentioned_issues'
+
+- content_for :ready_function do
+  :plain
+    var anchor = document.location.hash;
+    var result = [];
+    if (result = anchor.match(/^#diff_([\d]+)_n\d+$/)) {
+      var file_index = result[1];
+      $('.nav-tabs:not(.disable-link-generation) .nav-link[href="#changes"]').tab('show');
+
+      $('#revision_details_' + file_index).attr('open', 'open');
+
+      document.location.hash = anchor;
```

Comment:
Is it necessary to set the `location.hash`?

Reply:
A couple of days ago we introduced handling the event shown in tabs [here](https://github.com/openSUSE/open-build-service/blob/321a6cabcd436ff54b500563c30d985f8a67b1f7/src/api/app/assets/javascripts/webui/nav_tabs.js#L11-L19).

`document.location.hash` gets modified in line 86, when showing the tab:
```javascript
$('.nav-tabs:not(.disable-link-generation) .nav-link[href="#changes"]').tab('show');
```

If the line `document.location.hash = anchor;` is removed, the URL anchor part changes to `#tab-pane-changes`.

Reply:
Ah, I see. Okay